frontend/src/newConsole/components/newLogos/LogoSidbar.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
```

Comment:
Typo in filename: `LogoSidbar` -> `LogoSidebar`

Reply:
Thanks